### ISSUE TITLE
The training process CANNOT CONVERGE

### ISSUE THREAD

**Issue by yscheng04** [Intern-Nexus/gs-lrm-unofficial#1]
The training loss looks like this... Though the training process is far from finish, it looks like it's unlikely to converge and reach a good performance.

![Image](https://github.com/user-attachments/assets/a618f2fe-e3f4-4fb8-9b44-2c772ee7109a)

I haven't changed any config setting concerning training, and I use per GPU batch size equal to 4. I wonder whether it is normal such that the training process is very instable and is not very likely to converge or there's something going wrong.

**Comment by yscheng04** [comment]
BTW, I train the model on two RTX A6000 GPU. Could there be some problems with multi-GPU training?

**Comment by yscheng04** [comment]
Update：I gave up using multi-GPU training and everything is OK. Using batch_size = 2 needs far more training steps, but it can still converge and reach a decent performance.
